lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -70,6 +70,9 @@ export class AuthorizationCodeClient extends BaseClient {
         const reqTimestamp = TimeUtils.nowSeconds();
         const response = await this.executeTokenRequest(this.authority, request);
 
+        // Retrieve requestId from response headers
+        const requestId = response.headers && response.headers[HeaderNames.X_MS_REQUEST_ID] ? response.headers[HeaderNames.X_MS_REQUEST_ID] : undefined;
```

Comment:
I think you can leverage optional chaining here:

```suggestion
        // Retrieve requestId from response headers
        const requestId = response.headers?[HeaderNames.X_MS_REQUEST_ID];
```

If response.headers doesn't exist it'll return `undefined`. Then if it does exist it will either return the `HeaderName.X_MS_REQUEST_ID` header if present or `undefined` if not present.